packages/metal-dom/src/events.js

```diff
@@ -2,6 +2,54 @@
 
 import { registerCustomEvent } from './dom';
 import features from './features';
+import * as KEYMAP from './keyConstants';
+
+const REGEX_EVENT = /^(key+(down|up|press)):((,?[a-z])+)/;
+
+/**
+ * Returns the relative keyCode from each key name in an array.
+ * @param {Array<string>} keyNames An array with the key names.
+ */
+function convertKeynamesToKeyCode_(keyNames) {
+  let keys = {};
+
+  keyNames.forEach((value, index) => {
```

Comment:
It's best to avoid using `forEach` in places that may be called multiple times by the user, to avoid allocating temporary anonymous functions like this one.